src/test/java/no/nav/tag/kontaktskjema/gsak/GsakSchedulerTest.java

```diff
@@ -0,0 +1,26 @@
+package no.nav.tag.kontaktskjema.gsak;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GsakSchedulerTest {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @Test
+    public void skalFeileVedLagringAvKontaktskjemaMedForhandsdefinertId() throws InterruptedException {
+        Thread.sleep(1000);
+        assertThat(jdbcTemplate.queryForObject("SELECT COUNT(*) FROM SHEDLOCK", Integer.class), is(1));
```

Comment:
Denne testen er veldig enkel, og sjekker bare at det finnes en rad i shedlock-tabellen, som i praksis betyr at en shedlock-jobb har kjørt minst én gang. Dette er en dårlig fremgangsmåte, siden vi nå også er avhengige av oppsettet for hvor ofte jobben kjører, og må vente til vi vet at det har skjedd. Bør egentlig gjøres på en bedre måte, men får iallfall vist at dette initielle oppsettet fungerer og at shedlock blir benyttet.